component/file_cache/file_cache_linux_test.go

```diff
@@ -62,7 +62,7 @@ func (suite *fileCacheLinuxTestSuite) SetupTest() {
 	rand := randomString(8)
 	suite.cache_path = common.JoinUnixFilepath(home_dir, "file_cache"+rand)
 	suite.fake_storage_path = common.JoinUnixFilepath(home_dir, "fake_storage"+rand)
-	defaultConfig := fmt.Sprintf("file_cache:\n  path: %s\n  offload-io: true\n  timeout-sec: 0\n\nloopbackfs:\n  path: %s", suite.cache_path, suite.fake_storage_path)
+	defaultConfig := fmt.Sprintf("file_cache:\n  path: %s\n  offload-io: true\n  timeout-sec: 1\n\nloopbackfs:\n  path: %s", suite.cache_path, suite.fake_storage_path)
```

Comment:
Why in the tests do we need to bump the timeout-sec?

Reply:
This was just a choice for convenience. I want to emphasize testing with a non-zero timeout, since the zero timeout is a special case.

Reply:
Now that I've merged in the change to the way zero timeout eviction works, the way it evicts is so aggressive that it's hard to keep any files in cache long enough to complete a test.